editor/src/components/inspector/sections/layout-section/self-layout-subsection/gigantic-size-pins-subsection.tsx

```diff
@@ -587,18 +587,6 @@ export const GiganticSizePinsSubsection = React.memo((props: GiganticSizePinsSub
 
   return (
     <>
-      {unless(
-        isFeatureEnabled('Nine block control'),
-        <WidthHeightRow
```

Comment:
I think this now makes this component unused right?

Reply:
yeah I shall remove it probably, altough it _is_ containing the missing code that allows width and height pinning... maybe I should resurrect just that part and remove the others

Reply:
<img width="260" alt="image" src="https://github.com/concrete-utopia/utopia/assets/2226774/36866b4c-df9e-4b02-a591-9dd121c2e4eb">

@maltenuhn wdyt how wonky this looks? on master we are missing the two pin buttons for width and height, with this change I could put them back in an empty row. it looks unprofessional, but today IMO is even worse because we've simply lost these buttons and setting ie right, bottom, width, height pins is a pain today